.gitignore

```diff
@@ -59,6 +60,8 @@ target/
 customize.yml
 security/
 hosts
+jupyterhub_cookie_secret
+group_vars/jupyterhub_hosts
```

Comment:
For these jupyterhub related items make a new section with a comment explaining they are related to jupyterhub.